pkg/component/extensions/operatingsystemconfig/original/components/kubelet/component_test.go

```diff
@@ -69,7 +69,7 @@ var _ = Describe("Component", func() {
 				"blub": "bar",
 			}
 
-			cliFlags := CLIFlags(ctx.KubernetesVersion, ctx.NodeLabels, ctx.CRIName, ctx.Images["pause-container"], ctx.KubeletCLIFlags)
+			cliFlags := CLIFlags(ctx.KubernetesVersion, ctx.NodeLabels, ctx.CRIName, ctx.Images["pause-container"], ctx.KubeletCLIFlags, false)
```

Comment:
Would it make sense to test the new option?

Reply:
Test added.